prov/efa/src/rxr/rxr_pkt_type_req.c

```diff
@@ -853,6 +886,18 @@ ssize_t rxr_pkt_proc_matched_rtm(struct rxr_ep *ep,
 		rx_entry->cq_entry.len = rx_entry->total_len;
 
 	pkt_type = rxr_get_base_hdr(pkt_entry->pkt)->type;
+
+	if (pkt_type >= RXR_DC_REQ_PKT_BEGIN &&
+	    pkt_type <= RXR_BASELINE_DC_REQ_PKT_END)
+		rx_entry->fi_flags |= RXR_DELIVERY_COMPLETE_REQUESTED;
+
+	if (pkt_type == RXR_LONG_MSGRTM_PKT || pkt_type == RXR_LONG_TAGRTM_PKT)
+		rx_entry->tx_id = rxr_get_long_rtm_base_hdr(pkt_entry->pkt)->tx_id;
+	else
+		rx_entry->tx_id = rxr_get_dc_eager_medium_rtm_base_hdr(pkt_entry->pkt)->tx_id;
+
+	rx_entry->msg_id = rxr_get_rtm_base_hdr(pkt_entry->pkt)->msg_id;
+
```

Comment:
get ```rxr_entry->addr = pkt_entyr->addr ``` here, so you do not need to do it later in ```rxr_pkt_handle_data_copied```

Reply:
There is no need to do it here. `rx_entry->addr` is already defined in `rxr_pkt_get_msgrtm_rx_entry` and `rxr_pkt_get_tagrtm_rx_entry`